agent/util-scripts/test-bin/node_exporter

```diff
@@ -1 +1 @@
-mock-cmd
\ No newline at end of file
+dcgm-exporter
```

Comment:
Hmm, not to lob random monkey wrenches or anything; but I could see some case for renaming the mock from `dgcm-exporter` to something more generic given these changes.

Reply:
Since the currently-named `dgcm-exporter` is used by both `node_exporter` and `pmcd`, it _does_ seem like it ought to have a more generic name.

My question is, why are we maintaining `agent/util-scripts/test-bin/dcgm-exporter` separate from `agent/util-scripts/test-bin/mock-cmd`?  They seem similar enough that they could be merged, and then all of this magic would be in one place (and we wouldn't have to worry about naming the second one of these).